build_daemon/lib/data/client_options.dart

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Marker interface for any client-related information to be passed to builder.
+abstract class ClientOptions {}
```

Comment:
Is this necessary anymore? We can always add it in the future but I don't think it will be used both internally and externally given the changes to the build_target.